### DISCUSSION TITLE
CSVReader.java dosya ismi

### DISCUSSION THREAD

**Discussion by ozkanseyyarer** [gusanmaz/BMB212-Algorithms-2223#7] [Q&A] [answered]
Ödev de "CSVReader.java" isimli bir dosya oluşturun denilmiş. İçerisine yazacağımız kodlar da altında verilmiş bu kodların class ismi "CsvReader" olarak verilmiş. Dosya ismi ile class ismi aynı olmadığından program hata vermekte. Hangisini değiştirmeliyiz

**Comment by gusanmaz** [comment, marked as answer]
Kodda sinif ismine dikkat etmemisim. `README.md`'deki bu hata duzeltildi. Dosyanizin ismi `CSVReader.java`olacak.

**Reply by gusanmaz** [reply]
Koddaki sınıf ismi CSVReader olarak değiştirildi. Güncel Readme dosyasındaki kodu kullanabilirsiniz.

**Reply by ismailcanvarli** [reply]
teşekkürler.

**Comment by ismailcanvarli** [comment]
Başarılar.
